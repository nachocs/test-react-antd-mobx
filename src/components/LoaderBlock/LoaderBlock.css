.LoaderBlock {
  align-items: center;
  display: flex;
  flex-direction: row;
  justify-content: center;
  height: 100%;
}
