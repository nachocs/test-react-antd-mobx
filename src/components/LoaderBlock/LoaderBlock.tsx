import { Spin } from 'antd';
import * as React from 'react';

import './LoaderBlock.css';

export const LoaderBlock: React.FC = () => {
  return (
    <div className="LoaderBlock">
      <Spin />
    </div>
  );
};
