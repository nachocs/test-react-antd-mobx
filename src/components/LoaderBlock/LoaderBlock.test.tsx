import { shallow } from 'enzyme';
import * as React from 'react';

import { LoaderBlock } from './LoaderBlock';

it('LoaderBlock should be defined', () => {
  expect(LoaderBlock).toBeDefined();
});

it('LoaderBlock should render spinner', () => {
  const wrapper = shallow(<LoaderBlock />);
  expect(wrapper).toMatchSnapshot();
});
