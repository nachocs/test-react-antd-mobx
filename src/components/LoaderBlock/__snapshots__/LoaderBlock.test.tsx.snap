// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderBlock should render spinner 1`] = `
<div
  className="LoaderBlock"
>
  <Spin
    size="default"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
